packages/web/src/components/organisms/UserProfile/index.tsx

```diff
@@ -0,0 +1,108 @@
+import React, { useCallback, useState } from 'react'
+import { Button, Form, Input, Col, Row, Space } from 'antd'
+import { getAccountAddress } from 'src/fixtures/wallet/utility'
+import { useEffectAsync } from 'src/fixtures/utility'
+import { useGetUser, usePostUser } from 'src/fixtures/dev-for-apps/hooks'
+import { sign } from 'src/fixtures/wallet/utility'
+import styled from 'styled-components'
+
+interface Props {}
+
+const Container = styled.div`
```

Comment:
You can use the already defined `Container` component instead of the new styled-components! :)

Reply:
Sure!

fix in 85c078ec0191ddeca6abc0424677105211ba2f95